app/javascript/.storybook/main.js

```diff
@@ -86,6 +86,25 @@ module.exports = {
   },
   babel: async (options) => ({
     ...options,
-    plugins: [...options.plugins, 'inline-react-svg'],
+    plugins: [
+      ...options.plugins,
+      [
+        'inline-react-svg',
+        {
+          svgo: {
+            plugins: [
+              {
+                name: 'preset-default',
+                params: {
+                  overrides: {
+                    removeViewBox: false,
+                  },
+                },
+              },
+            ],
+          },
+        },
+      ],
+    ],
```

Comment:
Our icons didn't work properly in Storybook and generally in preact context. This should fix it. SVGO library was removing `viewBox` attribute by default and it was preventing icons from being resized. We never needed resized icons but for pills we do. Hence the update.

Reply:
Thank you @aitchiss for help!